packages.yml

```
@@ -20,8 +26,8 @@ packages:
   - package: fivetran/pinterest
     version: [">=1.1.0", "<1.2.0"]
 
-  - package: fivetran/reddit_ads
-    version: [">=1.1.0", "<1.2.0"]
+  # - package: fivetran/reddit_ads
+  #   version: [">=1.3.0", "<1.4.0"]
```

```suggestion
  - package: fivetran/reddit_ads
    version: [">=1.3.0", "<1.4.0"]
```
Once Reddit Ads is live